config/application.rb

```diff
@@ -52,5 +52,8 @@ class Application < Rails::Application
 
     # Our weeks in analytics should start on Monday
     Groupdate.week_start = :mon
+
+    # Let the app know what version we're running (REVISION is written by Capistrano on deploy)
+    config.x.current_revision = File.exist?("#{Rails.root}/REVISION") ? File.read("#{Rails.root}/REVISION").strip : "UNKNOWN"
```

Comment:
I stole that `x` from [the Rails Guide](http://guides.rubyonrails.org/configuring.html#custom-configuration). Is there a better way to do it?